tests/test_api.py

```diff
@@ -0,0 +1,90 @@
+import unittest
+
+try:
+    from .test_genius import genius
+except ModuleNotFoundError:
+    from test_genius import genius
+
+
+class TestAPI(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print("\n---------------------\nSetting up API tests...\n")
+
+    def test_account(self):
+        msg = ("No user detail was returned. "
+               "Are you sure you're using a user access token?")
+        r = genius.account()
+        self.assertTrue("user" in r, msg)
```

Comment:
I get a 401 Client Error on this test. Does it run successfully for you?

```
401 Client Error: Unauthorized for url: https://api.genius.com/account
This endpoint requires a valid user token.
```

Reply:
Yes, all tests run successfully for me ([this link](https://travis-ci.com/github/Allerter/LyricsGenius/builds/198647256) is the build for your last commit on my repo). I think Genius invalidates user tokens after new ones are generated, but I could be wrong about this.
Please run this code to generate a token and see if it works. If it did for the `genius.account()`, then it should work for the other endpoints like `upvote_annotations` as well:
```python
auth = OAuth2(GENIUS_CLIENT_ID,
              GENIUS_REDIRECT_URI,
              scope='all',
              client_only_app=True)
token = auth.prompt_user()
print(token)
genius = Genius(token)
print(genius.account().keys())
```

Reply:
Thanks for the explanation! Updating my token fixed the test.